samples/testing/integration/src/main.c

```diff
@@ -6,7 +6,12 @@
 
 #include <ztest.h>
 
-static void assert_tests(void)
+/**
+ * @brief Test Asserts
+ *
+ * This test verifies asserts
```

Comment:
This test doesn't verify that negative assertions cause a runtime error, only that positive assertions don't.  So the description of this test should mention that, right?

Reply:
that just complicates things to be honest, this is just to demonstrate the usage in a test, not the test itself. I modified it a bit.